__tests__/homedelivery/exporter.js

```diff
@@ -59,7 +59,7 @@ function verify (done, expectedError, expectedResponse, expectedFileName) {
             done.fail(err)
             return
           }
-          expect(expectedContents).toEqual(mockOutput)
```

Comment:
this is just an error I noticed, it doesn't really change anything other than the messaging displayed when the test fails